README.md

```diff
@@ -13,6 +13,13 @@ $ meteor npm install svelte@<version>
 
 **Important:** The version of the `svelte` npm package should match the version of `svelte:compiler`.
 
+To support legacy browsers, the package must be recompiled. This will be done automatically by Meteor by creating a symbolic link in the `imports` directory:
+
+```
+$ cd imports
+$ ln -s ../node_modules/svelte .
```

Comment:
Or you can put it in your `package.json` under `meteor.nodeModules.recompile`.
See https://guide.meteor.com/using-npm-packages.html#recompile for more details.